src/wallet/api/wallet2_api.h

```diff
@@ -1144,6 +1149,545 @@ struct Wallet
+// TODO : accept_func with wallet2::multisig_tx_set
+    /**
+    * brief: loadMultisigTxFromFile - load a multisig transaction set from a file
+    * param: multisig_tx_str -
+    * outparam: exported_txs -
+    * return: true if succeeded
+    */
+//    virtual bool loadMultisigTxFromFile(const std::string &filename, PendingTransaction &exported_txs, std::function<bool(const multisig_tx_set&)> accept_func) const = 0;
+// TODO : accept_func with wallet2::multisig_tx_set
+    /**
+    * brief: signMultisigTxFromFile - load a multisig transaction set from a file, sign it and store to a file
+    * param: filename - load from and store to file with this filename
+    * outparam: txids - transaction ids of signed transactions
+    * param: accept_func - callback function to verify the transaction
+    * return: true if succeeded
+    */
+//    virutal bool signMultisigTxFromFile(const std::string &filename, std::vector<crypto::hash> &txids, std::function<bool(const multisig_tx_set&)> accept_func) const = 0;
+    /**
+    * brief: getFeeMultiplier -
+    * param: priority -
+    * param: fee_algorithm -
+    * return: fee multiplier
+    */
+    virtual std::uint64_t getFeeMultiplier(std::uint32_t priority, int fee_algorithm) const = 0;
+    /**
+    * brief: getBaseFee -
+    * return: dynamic base fee estimate if using dynamic fee, else FEE_PER_KB
+    */
+    virtual std::uint64_t getBaseFee() const = 0;
+    /**
+    * brief: getMinRingSize - get the minimum allowed ring size
+    * return: min ring size
+    */
+    virtual std::uint64_t getMinRingSize() const = 0;
+    // TODO : Should we make this static?
+    /**
+    * brief: adjustMixin - adjust mixin to fit into min and max ring size
+    * param: mixin -
+    * return: adjusted mixin
+    */
+    virtual std::uint64_t adjustMixin(std::uint64_t mixin) const = 0;
+    /**
+    * brief: adjustPriority - adjust priority depending on how "full" last N blocks are
+    * param: priority -
+    * return: adjusted priority
+    * note: sets status error on fail
+    */
+    virtual std::uint32_t adjustPriority(std::uint32_t priority) const = 0;
+    // QUESTION : Can anyone help with these comments? I looked at the command descriptions in `simplewallet::simplewallet()`, but I think they're too vague and imo they don't really help understand what these functions do.
+    /**
+    * brief: unsetRing -
+    * param: key_images -
+    * param: tx_id -
+    * return: true if succeeded
+    * note: sets status error on fail
+    */
+    virtual bool unsetRing(const std::vector<std::string> &key_images) = 0;
+    virtual bool unsetRing(const std::string &tx_id) = 0;
+    /**
+    * brief: findAndSaveRings -
+    * param: force - force save (default: true)
+    * return: true if succeeded
+    * note: sets status error on fail
+    */
+    virtual bool findAndSaveRings(bool force = true) = 0;
```

Comment:
Maybe we could put up this issue for a separate discussion, but to me it seems that nobody at all uses any of that "blackballing" stuff anymore, and we could just drop everything connected with it. Maybe it even became victim of "rot", i.e. if you tried to actually use you would find out that it did not care for some changes that happened since it was introduced and does not fully work anymore.

Reply:
This function is unrelated to blackballing, see: https://github.com/monero-project/monero/commit/5f146873c58f39632e26c5edbf2f618cacbd76a5. It should probably not be exposed in the API as it can leak all wallet txids to the daemon. Fork re-use mitigation might be obselete with FCMP++, so perhaps it can considered for removal, too? For discussion on removing blackballing, see #8758.

Reply:
Oh thanks for pointing that out, I wasn't aware.
Will remove it then.